src/serviceWorker.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
Why do you have eslint disabled for this file?

Reply:
This is because I was getting these issues
<img width="519" alt="screenshot 2019-01-22 at 11 16 34" src="https://user-images.githubusercontent.com/36000749/51521287-3c95de00-1e37-11e9-819e-ae8a5761a519.png">
<img width="535" alt="screenshot 2019-01-22 at 11 16 29" src="https://user-images.githubusercontent.com/36000749/51521289-3dc70b00-1e37-11e9-9a3c-3d354616b24c.png">